corehq/apps/change_feed/topics.py

```diff
@@ -6,6 +6,7 @@
 META = 'meta'
 
 # new models
+CASE_SQL = 'case-sql'
```

Comment:
I'm not 100% sure it will work about, but I am about 80% sure. I'll get rid of the notion of a data source if it doesn't